Tests/DoublePinchTests/TappableButtonTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TappableButtonTests.swift
+//  DoublePinch
+//
+
+import SwiftUI
+import TappableButton
+import Testing
+
+struct TappableButtonTests {
+    
+    @MainActor
+    @Test("Action can be performed programmatically", .timeLimit(.minutes(1)))
+    func performAction() async throws {
+        try await confirmation { confirm in
+            let button = SwiftUI.Button("") { confirm() }
```

Comment:
Could we declare the action closure separately and confirm it's the same as the one we extract from the button? Or compare the outputs of the actions if it's not possible to check equality with closures?